src/components/Display/Display.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import PropTypes from 'prop-types'
```

Comment:
```suggestion
import { string } from 'prop-types'
```
For prop-types, we'll usually import individual modules rather than the entire `prop-types` library.